releases/release-1.20/meeting-updates/2020-11-16.md

```diff
@@ -0,0 +1,172 @@
+# Meeting Updates 2020-11-16
+
+Cutoff Time: 2020-11-16 10:30:00 PST (Times in a few other timezones as well)
+Repository sync responsibility: Jeremy / @jerickar
+
+## Enhancements
+
+Author: Name / Slack Handle
+Current Status: Freen
```

Comment:
nitpick:
```suggestion
Current Status: Green
```

Reply:
yeah I didn't proof read, I just cut/paste. I don't like a formatting thing fixing that up too.